src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/PullRequestPostAnalysisTask.java

```diff
@@ -92,14 +92,15 @@ public void finished(Context context) {
             return;
         }
         ProjectAlmSettingDto projectAlmSettingDto = new ProjectAlmSettingDto();
+        String enablePrDecoration = Optional.ofNullable(System.getenv("PR_DECORATION_ENABLED")).orElse("false");
```

Comment:
This is a system variable and only works for Sonar globally.
What I thought could be useful was a way so a single project can turn off the feature through a parameter sent by the scanner (default true)

Reply:
By "through parameter sent by the scanner"
Do you mean something like

In Jenkinsfile (per service repo) we could override it with
`PR_DECORATION_ENABLED=false`

And from the jenkins library we will introduce something like
`./gradlew -DPR_DECORATION_ENABLED=env.PR_DECORATION_ENABLED` (env default value is true).